seed/management/commands/create_default_user.py

```diff
@@ -52,13 +52,13 @@ def handle(self, *args, **options):
 
             if options['usertype'] == 'superuser':
                 u = User.objects.create_superuser(
-                    options['username'],
+                    options['username'].lower(),
```

Comment:
oh, very nice if using the command line to create the user. We probably still have users with both lower case and mixed case names in the database!

Reply:
Yeah, more than a few times I used capital letters in my email address with that command and was prevented from logging in until I updated the database value 🤦‍♂️